.github/prod-cluster/backend-config.yaml

```diff
@@ -1,4 +1,4 @@
-# Copyright 2021 Google LLC
+# Copyright 2022 Google LLC
```

Comment:
Copyright headers should reflect the earliest year that the copyright header was instantiated (in this case `2021`). You can either:
- Leave the older year; or
- Append a newer year while keeping the older one (e.g. `2021-2022`)

I prefer the former since it's less maintenance.

Reply:
Good to know, done.